public/app/features/alerting/unified/components/notification-policies/Policy.test.tsx

```diff
@@ -0,0 +1,257 @@
+import { screen, render, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { noop } from 'lodash';
+import React from 'react';
+import { Router } from 'react-router-dom';
+
+import { locationService } from '@grafana/runtime';
+import { contextSrv } from 'app/core/core';
+import {
+  AlertmanagerGroup,
+  MatcherOperator,
+  ObjectMatcher,
+  RouteWithID,
+} from 'app/plugins/datasource/alertmanager/types';
+import { ReceiversState } from 'app/types/alerting';
+
+import { mockAlertGroup, mockAlertmanagerAlert, mockReceiversState } from '../../mocks';
+import { GRAFANA_RULES_SOURCE_NAME } from '../../utils/datasource';
+
+import { Policy } from './Policy';
+
+beforeAll(() => {
+  userEvent.setup();
+});
+
+describe('Policy', () => {
+  beforeAll(() => {
+    jest.spyOn(contextSrv, 'hasPermission').mockReturnValue(true);
+  });
+
+  it('should render a policy tree', async () => {
+    const onEditPolicy = jest.fn();
+    const onAddPolicy = jest.fn();
+    const onDeletePolicy = jest.fn();
+    const onShowAlertInstances = jest.fn(
+      (alertGroups: AlertmanagerGroup[], matchers?: ObjectMatcher[] | undefined) => {}
+    );
+
+    const routeTree = mockRoutes;
+
+    renderPolicy(
+      <Policy
+        routeTree={routeTree}
+        currentRoute={routeTree}
+        alertManagerSourceName={GRAFANA_RULES_SOURCE_NAME}
+        onEditPolicy={onEditPolicy}
+        onAddPolicy={onAddPolicy}
+        onDeletePolicy={onDeletePolicy}
+        onShowAlertInstances={onShowAlertInstances}
+      />
+    );
+
+    // should have default policy
+    const defaultPolicy = screen.getByTestId('am-root-route-container');
+    expect(defaultPolicy).toBeInTheDocument();
+    expect(within(defaultPolicy).getByText('Default policy')).toBeVisible();
+
+    // should be editable
+    const editDefaultPolicy = within(defaultPolicy).getByRole('button', { name: 'Edit' });
+    expect(editDefaultPolicy).not.toBeDisabled();
+
+    await userEvent.click(editDefaultPolicy);
+    expect(onEditPolicy).toHaveBeenCalledWith(routeTree, true);
+
+    // click "more actions" and check if we can delete
+    await userEvent.click(within(defaultPolicy).getByTestId('more-actions'));
+    expect(await screen.queryByRole('menuitem', { name: 'Delete' })).not.toBeInTheDocument();
+
+    // default policy should show the metadata
```

Comment:
either missing assertion or extra comment here 😄

Reply:
This comment is extra, it basically says "we're going to make some assertions on metadata of the default policy from here on" but I get why that would be confusing since the next line is another comment about what we're checking specifically 😅